pysat/tests/test_ssnl_occur_prob.py

```diff
@@ -0,0 +1,112 @@
+"""
+tests the pysat occur_prob object and code
+"""
+
+import numpy as np
+import sys
+
+from nose.tools import assert_raises, raises
+import pandas as pds
+import tempfile
+
+import pysat
+from pysat.ssnl import occur_prob
+
+
+class TestBasics():
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
+        orbit_info = {'index': 'longitude', 'kind': 'longitude'}
+        self.testInst = pysat.Instrument('pysat', 'testing',
+                                         clean_level='clean',
+                                         orbit_info=orbit_info)
+        self.testInst.bounds = (pysat.datetime(2008, 1, 1),
+                                pysat.datetime(2008, 3, 31))
```

Comment:
Why not test for a month instead of 3 months?  Will probably speed things up.

Reply:
I'll give it a try.  Takes about 41 seconds now.

Reply:
Need to increase the tolerance for one test from 1.0e-2 to 2.0e-2, but cuts the tests for this file down to 15 seconds.  Happy to go either way on this.